Core/src/TrackFitting/MbfSmoother.cpp

```diff
@@ -55,10 +61,8 @@ void MbfSmoother::visitMeasurement(const InternalTrackState& ts,
     typename TrackStateTraits<kMeasurementSize, true>::CalibratedCovariance
         calibratedCovariance{measurement.calibratedCovariance};
 
-    // Measurement matrix
-    const MeasurementMatrix H =
-        measurement.projector
-            .template topLeftCorner<kMeasurementSize, eBoundSize>();
+    // Projector matrix
+    const ProjectorMatrix H = subspaceHelper.projector();
```

Comment:
Same here: should this eventually go away or do we need the matrix?

Reply:
I think this can all be replaced with the subspace projection in a second iteration